client/app/user/profile.html

```diff
@@ -3,6 +3,9 @@
   <form>
     <input type="text" ng-model="hero.author" /></h2>
     <textarea ng-model="hero.bio" />
+      <select class="u-full-width" id="exampleRecipientInput">
+        <option ng-repeat="theme in themes"> {{ theme }} </option>
```

Comment:
should  {{ theme }}      {{ theme.title }} 
it will depend on the data returned by Themes service